docs/node-operators/how-to/network-upgrade-from-ops-perspective.md

```diff
@@ -1,33 +1,55 @@
 # Upgrade procedure from the OPS perspective
 
-## 0. Assumptions
+## Assumptions
 
-Before you start, you have to prepare and keep in mind a few things:
+**NOTE**: Below instruction is prepared for Debian-like Linux operating systems. We also assume you are already running a validator node.
 
-**NOTE**: Below instruction is prepared for Debian-like Linux operating systems.
+Before you start, you have to prepare and keep in mind a few things:
 
 - `<VEGA-USER>`; the Linux user that runs vega, e.g., `vega`
 - `<USER-HOME>`; path to the `<VEGA-USER>` home directory, e.g., `/home/vega`
 - `<VEGA-NETWORK-HOME>`; the home path to the vega network, e.g., `/home/vega/vega_home`
+- `<DATA-NODE-HOME>`; the home path to the data-node, the home can be safely shared with vega core, e.g., `/home/vega/data_node_home`
 - `<TENDERMINT-HOME>`; the tendermint home path, e.g., `/home/vega/tendermint_home`
 - `<VEGAVISOR-HOME>`; the vegavisor home path, e.g., `/home/vega/vegavisor_home`
 - `<BACKUP-FOLDER>`; the folder where you store backups, e.g., `/home/vega/backups`
 - `<VISOR-BIN>`; the path to the vega visor binary, e.g., `/home/vega/bin/visor`
 - `<VEGA-BIN>`; the path to the vega core binary, e.g., `/home/vega/bin/visor`
+- `<CHAIN-ID>`; new chain id for network, it is required to pass as an argument for data-node, e.g., current chain id on mainnet is: `vega-mainnet-0009`
 
 We will refer to the above paths in the following article. The sample paths given above are just examples. We recommend setting your paths that align with the conventions adopted in your company.
 
 # Upgrade steps
 
-## 1. Prepare genesis file
+## 1. Stop the network
+
+At this point, validators choose the checkpoint, which will be loaded into the network during the next restart, and stop the network as soon as everyone agrees on the selected checkpoint. The reason to quickly stop the network is to avoid producing more checkpoints and missing transactions after restart(the transactions executed between selected checkpoint is produced, and the network is stopped)

```

Comment:
```suggestion
At this point, validators need to choose and agree on the checkpoint that will be loaded into the network during the next restart, and stop the network as soon as everyone agrees on the selected checkpoint. The reason to quickly stop the network is to avoid producing more checkpoints and missing transactions after restart (the transactions executed between when the selected checkpoint is produced, and the network is stopped).
```